t/run/switches.t

```diff
@@ -301,8 +301,8 @@ is runperl(stderr => 1, prog => '#!perl -M'),
         skip "Win32 miniperl produces a default archname in -v", 1
 	  if $^O eq 'MSWin32' && is_miniperl;
         my $v = sprintf "%vd", $^V;
-        my $ver = $Config{PERL_VERSION};
-        my $rel = $Config{PERL_SUBVERSION};
+        my $ver = $Config{PERL_VERSION_MINOR};
+        my $rel = $Config{PERL_VERSION_PATCH};
```

Comment:
Probably makes sense to rename these to `$ver_minor` and `$ver_patch` for consistency